ci/test/pipeline.template.yml

```diff
@@ -670,7 +670,7 @@ steps:
     plugins:
       - ./ci/plugins/mzcompose:
           composition: feature-benchmark
-          args: [--scenario=KafkaUpsertUnique]
+          args: [--scenario=KafkaUpsertUnique --other-tag=common-ancestor]
```

Comment:
I think the normal CI should be left as it is, that is, it will run a dummy feature-benchmark comparison between HEAD and HEAD. This is to make sure that the feature-benchmark scripts were not fundamentally broken by the commit, it is not about detecting a performance regression.

The actual performance regressions are detected in Nightly, which should run agains tthe `common ancestor" if on a PR and against "latest" otherwise. This way, if there is a performance regression, merging is not immediatey blocked, as it would have been if the main CI was failing.

Reply:
✅